src/form.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+
+export default class Form extends React.Component {
+  constructor(props) {
+    super(props)
+    this.handleSubmit = this.handleSubmit.bind(this)
+  }
+  handleSubmit(event) {
+    event.preventDefault()
+    const newCardData = new FormData(event.target)
+    const newCard = {
+      question: newCardData.get('question'),
+      answer: newCardData.get('answer')
+    }
+    this.props.submit(newCard)
+    event.target.reset()
+  }
+  render() {
+    const style = {
```

Comment:
It's best to define static `styles` outside of your `render` method. You can move this variable above your class definition.